tests/Table.spec.js

```diff
@@ -14,6 +14,17 @@ describe('Table.Basic', () => {
 
   describe('renders correctly', () => {
     it('basic', () => {
+      const wrapper = mount(
+        createTable({
+          prefixCls: 'test-prefix',
+          className: 'test-class-name',
+          direction: 'rtl',
```

Comment:
Should not change `basic` test with `rtl`. Follow `RTL` is good enough.

Reply:
That was a mistake.
I wanted to add  `direction: 'rtl'` to `RTL` test case.
.
Fixed.